webClient/src/app/editor/code-editor/monaco/monaco.component.ts

```diff
@@ -40,10 +44,35 @@ export class MonacoComponent implements OnInit, OnChanges {
   }
 
   ngOnInit() {
+    (<any>window).monaco.baseUrl = '/ZLUX/plugins/org.zowe.editor/web/';
+    this.monacoConfig = new MonacoConfig();
+    let options = Object.assign({}, this.monacoConfig.defaultOptions, this.options);
+    console.log('options to load=',options);
+    const hasModel = !!options.model;
```

Comment:
You're comparing truthy value, but isn't this comparing truthy of "options.model" ? Thus, would fail if options ever didn't exist?

Reply:
Yes it will
![image](https://user-images.githubusercontent.com/20528015/84626638-71368980-aeb3-11ea-985c-6533ba81ca64.png)
and if it doesn't matter because options will always be initialized, then that's fine & code complete given the context, but then isn't it safe to assume model will always exist too?

Reply:
That might explain some past errors.